core/domain/exp_jobs_one_off.py

```diff
@@ -535,3 +536,41 @@ def reduce(key, values):
         # Combine all values from multiple lists into a single list
         # for that error type.
         yield (key, list(set().union(*final_values)))
+
+
+class ImageDataMigrationJob(jobs.BaseMapReduceOneOffJobManager):
+    """ One-off job for migrating the images in the exploration
+        from the GAE to GCS
+    """
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [file_models.FileSnapshotContentModel]
+
+    @staticmethod
+    def map(file_snapshot_content_model):
+        instance_id = file_snapshot_content_model.get_unversioned_instance_id()
+        filetype = instance_id[instance_id.rfind(".") + 1:]
+
+        if filetype in ALLOWED_IMAGE_EXTENSIONS:
+            filename = instance_id[instance_id.rfind("/") + 1:]
+            exploration_id = (instance_id[1:])[:(instance_id[1:]).find("/")]
+            filepath = (instance_id[1:])[(instance_id[1:]).find("/") + 1:]
+            file_model = file_models.FileModel.get_model(
+                exploration_id, filepath, False)
+
+            if file_model and file_model.deleted:
+                # content = file_model.content
+                # fs = fs_domain.AbstractFileSystem(fs_domain.GcsFileSystem(
+                #   exploration_id))
+                # fs.commit(file_model.user_id, '%s/%s' %('images',
+                #   filename, content)
+                file_model.delete("ADMIN", "Delete the file", False)
```

Comment:
The third arg has a default value, so the name of the arg should be specified.

I.e. file_model.delete(..., ..., name_of_arg=False)

/cc @apb7 @kevinlee12 this is quite important, we should catch this.